multi-tenancy-observability/README.md

```diff
@@ -0,0 +1,270 @@
+
+### Future of monitoring
+
+We already have plans to migrate our monitoring stack to Grafana Mimir, which supports the same multi-tenancy mechanism as Loki. You can see the current status in this [epic](https://github.com/giantswarm/roadmap/issues/3039).
+
+## What we want to achieve
+
+We want to be able to segregate access to the observability data based on the user groups our customers have configured in their active directory.
+
+But we also want to provide flexibility to customers to allow them to isolate their data as they want.
+For instance, some customers may want to map:
+- 1 tenant to 1 namespace
+- 1 tenant to 1 cluster
+- multiple tenants to 1 application (e.g. ingress controllers)
+We would like to propose our customers the option of defining their tenants so that they can isolate the data as they want.
+
+__Disclaimer:__ We might change this flexibility later on based on product (e.g. project CR) or operational decisions ( e.g. too many tenants).
+
+In this document we will look at the multi-tenancy concept on the example of the logging implementation (with `Loki`) as this is where we already collected experience with it. But we expect to share the same logic across all of our observability stack components.
+
+Because the read and write path for the multi-tenancy is technically different and have different requirements, we decided to split it into different sections.
+
+## Read path
+
+### Current architecture for the read path
+
+Below is a graph exposing the current state of multi-tenancy:
+
+<img src="./assets/current-multi-tenancy.png" width="300" alt="Loki multi-tenancy on the read path">
+
+The read path queries `loki` providing an `X-Scope-OrgID` header which contains the identifier (the `read` tenant) of all clusters: the management cluster plus all workload clusters.
+
+### Configuring multi-tenancy on the read path
+
+We have asserted that we can retrieve users groups through the OAuth token that is received from Grafana when we enable OAuth-forwarding on the Loki datasource.
+
+Below is an example of OAuth token content coming from Dex:
+
+```json
+{
+  "iss": "https://dex.golem.gaws.gigantic.io",
+  "sub": "XXXXXXX",
+  "aud": "YYYYYYY",
+  "exp": 1705661375,
+  "iat": 1705659575,
+  "email": "people@giantswarm.io",
+  "email_verified": true,
+  "groups": [
+    "giantswarm-ad:giantswarm-admins",
+    "giantswarm-ad:GS Support - MS teams",
+    "giantswarm-ad:Giant Swarm Global",
+    "giantswarm-ad:GiantSwarm",
+    "giantswarm-ad:Giant Swarm EU",
+    "giantswarm-ad:Developers"
+  ],
+  "name": "People People"
+}
+```
+
+The component `loki-multi-tenant-proxy` which has now been renamed to `grafana-multi-tenant-proxy` should be configured with the mapping between groups of people and tenants
```

Comment:
Maybe it could be worth to explain in a short sentence why the name has changed.